includes/polling/netstats-snmp.inc.php

```diff
@@ -54,7 +54,7 @@
             $value = $data_array[0][$oid];
         }
         else {
-            $value = 'U';
+            $value = null;
```

Comment:
TBH, this entire loop can probably go. Its duplicating what is already done in includes/rrdtool.inc.php:rrdtool_update

This was changed as this will create the influx field as a string if it is the first data to be inserted. This is no good when the actual data if it is not defined is a numeric value.